jerry-core/ecma/base/ecma-helpers-string.c

```diff
@@ -1351,31 +1318,6 @@ ecma_is_string_magic_longpath (const ecma_string_t *string_p, /**< ecma-string *
 } /* ecma_is_string_magic_longpath */
 
 /**
- * Long path part of ecma_is_ex_string_magic
- *
- * Converts passed ecma-string to zt-string and
- * checks if it is equal to one of magic string
- *
- * @return true - if magic string equal to passed string was found,
- *         false - otherwise.
- */
-static bool
-ecma_is_ex_string_magic_longpath (const ecma_string_t *string_p, /**< ecma-string */
-                                  lit_magic_string_ex_id_t *out_id_p) /**< [out] external magic string's id */
```

Comment:
I think `ecma_is_ex_string_magic` and `ecma_is_ex_string_magic_longpath` are unnecessary and never really used. I agree with removing them.